pwiz_aux/msrc/utility/vendor_api/Bruker/TimsData.cpp

```diff
@@ -501,6 +559,45 @@ TimsDataImpl::TimsDataImpl(const string& rawpath, bool combineIonMobilitySpectra
                 ms2Time = frame->rt_;
                 ms2Intensities.push_back(frame->tic_);
             }
+
+            // Note contents of DiaFrameMsMsWindows, with scan range translated to 1/K0 range
+            if (oneOverK0ByScanNumberByCalibration_.size() == 1) // Only works if calibration doesn't change in run

```

Comment:
So I guess we'll fix multi-calibration files when we get one? :)

Reply:
Yeah, if and when.